build.gradle

```diff
@@ -148,6 +148,7 @@ dependencies {
         exclude module: "json"
     }
     compile 'com.fasterxml.jackson.datatype:jackson-datatype-joda:2.4.0'
+    compile 'com.grack:nanojson:1.1'
```

Comment:
If you want to be really involved in this, I will give some context:
The original problem was found during test phase of our Nakadi UI. It happens that we use json parser wihch is not fully json rfc comptible that was about multiple roots and arbitary symbols at the end of the json schema. The solution I provided for that was pretty simple just go through the ET schema and count the number of the open/close brackets if something left after num of brackets is 0 then it was considered as invalid schema (https://github.com/zalando/nakadi/pull/546/commits/45b8bad8b37852cedb78223f3f9e523452a58b2d#diff-0cc0d4406e77f42ca8a2ee3e3ce445e1R222). 
Suddenly, another case appeared where schema had trailing commas in the json arrays and objects which of course was not covered by the solution.

We found out several possible ways to solve it:
1. Change nothing, leave it as it is
	As we understand it is not critical for our application, because we still can work with this malformed json schema and we have higher priority things to accomplish. In this approach we do not care about users that much (they lived with that for more than one year), becasuse they will see unprocessable json schema in the UI, but concentrate on more important task.

2. Store only parsed json object
	Right now after parsing ET json schema we store provided json schema not the parsed one. This approache considered stroring schema in our database after it was parsed. The first downside is that not clear for user since he provided one ET schema, but at the end got other and the second there could be mistakes, when user provided json schema with multiple roots, but we stored only the one of them (it how parsing works now).
	
3. Our own implementation
	The approache is quite interesting, but very error prone in the way that json parsing has a lot of cases and probably we'll miss somethig, which will introduce real user affecting bugs.
	
4. Third party dependecy
	We have several third party libraries connected to our project, but non of them are support json parsing in a proper way. This approache is less error prone comparing to our own implementation and of course it can have their own bugs. We used http://seriot.ch/json/parsing.html to find out the parser which better suites for our needs and nanojson looks like it.
	
About number of dependencies:
Actually until this moment we have not had any direct dependecies on json parser they are depenencies of other libraries we use. As to my calculation it is about 3: jackson, JSON, json-smart.

To conclude, probably, there are other solutions to that problem. I choose this one to be more customer centric with less effort possible I agree it is not the best possible solution, but more pragmatic. In other case I would rather go for 1.